src/cmake/install.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -e
+source _common.sh
+
+ensure_apt_packages curl ca-certificates jq
+
+if [[ -z $VERSION || $VERSION == latest ]]; then
+  echo "🟪 Fetching latest CMake release..."
+  curl -fsSLo latest-release.json https://api.github.com/repos/Kitware/CMake/releases/latest
```

Comment:
Please do not use the GitHub API; we should not expect to get a version due to API limits.